modules/graphstudio/pages/write-queries.adoc

```diff
@@ -93,12 +92,13 @@ image:write-queries-run-install-query-with-param_3.3.png[image]
 
 == Run Configuration
 
-You can set runtime configurations of the GSQL query by clicking the small image:3.9.png[image] button to the right of the Run Query button, then a drop-down menu will appear with the btn:[Run Configuration] button.
+You can set runtime configurations of the GSQL query by clicking the small downward-pointing arrow button image:3.9.png[image] to the right of the Run Query button, then a drop-down menu will appear with the btn:[Run Configuration] button.
 
 image:write-queries-use-default-timeout_3.3.png[image]
 
 This opens the Run Configuration panel, where you can set the GSQL query timeout.
 By default, it uses the timeout specified with gadmin server commands in the TigerGraph configuration.
+See the xref:tigergraph-server:system-management:management-commands.adoc[TigerGraph Server Command Glossary] for more information.
```

Comment:
This is not sufficiently helpful.  I scanned the list of hundreds of configuration parameters. I can't tell which parameter I would set.